pkgs/development/python-modules/cx_freeze/default.nix

```
@@ -9,6 +9,8 @@ buildPythonPackage rec {
     sha256 = "44bbbcea3196b79da77cc22637cb28a825b51182d32209e8a3f6cd4042edc247";
   };
 
+  disabled = python.isPy2;
```

Same as in other https://github.com/NixOS/nixpkgs/pull/97766, use ``pythonOlder``
```suggestion
  disabled = pythonOlder "3.5";
```
https://github.com/marcelotduarte/cx_Freeze/blob/bfa89fc010b3c6ab6a578d0fb7c4524c56d974dc/setup.py#L12-L15